### PR TITLE
DPO Shopify Icons

### PR DESCRIPTION
**Pull request by linwor**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [ ] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

File: app/assets/images/payment_icons/dpo-xpay.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...EUBRL4f4HNtJ01U0pUWkM2OP3llZEcLL8GhsAgO4cXQd0GIN+EocAAAAAAAD6cBR+ fuOYOAAAAAAAgMTEIAAAAAAAgMTEIAAAAAAAgMTEIAAAAAAAgMTEIAAAAAAAgMTEIAAAAAAAgMTE IAAAAAAAgMTEIAAAAAAAgMT+HIPW9VV2iwAAAAAAAHTNZBAAAAAAAEBiYhAAAAAAAEBip2KQo+IA AAAAAADGcnoySBACAAAAAAB4ztlWUyJit3hWrdvnDwEAAAAAAOCt/wztNIlBiyAEAAAAAADQVKvT 2k4fEwcAAAAAAMA4xCAAAAAAAIDExCAAAAAAAIDExCAAAAAAAIDExCAAAAAAAIDOrOurtHqjEhG7 xRZq3a55MAAAAAAAQDIt489Pl8Wgb6IQAAAAAADAsStD0OKYOAAAAAAAgOdcHYKWO2LQHZsAAAAA AADgdyaDAAAAAAAAEhODAAAAAAAAEhODAAAAAAAAErslBrk3CAAAAAAA4BklIh778lq3574cAAAA AADgBk8PzTx6TJyJIQAAAAAAgGu5MwgAAAAAACAxMQgAAAAAACAxMQgAAAAAACAxMQgAAAAAACCx x2PQur7KbhEAAAAAACCBHjpIiYjd4tNq3fp7KQAAAAAAgBN6GYjpMgZ9E4UAAAAAAIDe9X4KmjuD AAAAAAAAEus6BrlPCAAAAAAA4DMmgwAAAAAAABITgwAAAAAAABITgwAAAAAAABIrETHc7mrdxntp AAAAAABgeOv6KqPtYcjJoBF/aAAAAAAAYGyj9olhj4kThAAAAAAAgLuM3CXcGQQAAAAAAJCYGAQA AAAAAJCYGAQAAAAAAPDG6FfXDB2D3BsEAAAAAADwXomItx8YWa1b3s0BAAAAAADNZB5ASR2DjohE AAAAAAAwpxlPHXNnEAAAAAAAQGJiEAAAAAAAMIUZp4IWMQgAAAAAACA3MQgAAAAAACCxKWPQrGNg AAAAAADAfEpETPu317rNu3kAAAAAAJjE7EMiU8egVkQlAAAAAAC4xuwhpwUxqCFRCAAAAAAA2hCB 2hGDbiYYAQAAAAAwM5HnZsuyfAHbJFaV5Kl48AAAAABJRU5ErkJggg==" transform="translate(2.276 1.414) scale(.02001)" overflow="visible"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/dpo-xpay.svg:1]: Please remove `<style type="text/css"/>`

File: app/assets/images/payment_icons/dpo-visa.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...tyle type="text/css"/><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" opacity=".07"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/><path d="M28.3 10.1H28c-.4 1-.7 1.5-1 3h1.9c-.3-1.5-.3-2.2-.6-3zm2.9 5.9h-1.7c-.1 0-.1 0-.2-.1l-.2-.9-.1-.2h-2.4c-.1 0-.2 0-.2.2l-.3.9c0 .1-.1.1-.1.1h-2.1l.2-.5L27 8.7c0-.5.3-.7.8-.7h1.5c.1 0 .2 0 .2.2l1.4 6.5c.1.4.2.7.2 1.1.1.1.1.1.1.2zm-13.4-.3l.4-1.8c.1 0 .2.1.2.1.7.3 1.4.5 2.1.4.2 0 .5-.1.7-.2.5-.2.5-.7.1-1.1-.2-.2-.5-.3-.8-.5-.4-.2-.8-.4-1.1-.7-1.2-1-.8-2.4-.1-3.1.6-.4.9-.8 1.7-.8 1.2 0 2.5 0 3.1.2h.1c-.1.6-.2 1.1-.4 1.7-.5-.2-1-.4-1.5-.4-.3 0-.6 0-.9.1-.2 0-.3.1-.4.2-.2.2-.2.5 0 .7l.5.4c.4.2.8.4 1.1.6.5.3 1 .8 1.1 1.4.2.9-.1 1.7-.9 2.3-.5.4-.7.6-1.4.6-1.4 0-2.5.1-3.4-.2-.1.2-.1.2-.2.1zm-3.5.3c.1-.7.1-.7.2-1 .5-2.2 1-4.5 1.4-6.7.1-.2.1-.3.3-.3H18c-.2 1.2-.4 2.1-.7 3.2-.3 1.5-.6 3-1 4.5 0 .2-.1.2-.3.2M5 8.2c0-.1.2-.2.3-.2h3.4c.5 0 .9.3 1 .8l.9 4.4c0 .1 0 .1.1.2 0-.1.1-.1.1-.1l2.1-5.1c-.1-.1 0-.2.1-.2h2.1c0 .1 0 .1-.1.2l-3.1 7.3c-.1.2-.1.3-.2.4-.1.1-.3 0-.5 0H9.7c-.1 0-.2 0-.2-.2L7.9 9.5c-.2-.2-.5-.5-.9-.6-.6-.3-1.7-.5-1.9-.5L5 8.2z" fill="#142688"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/dpo-visa.svg:1]: I'm curious about the reason to add this. We already have the visa icon: https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg

File: app/assets/images/payment_icons/dpo-tigopesa.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...+PiQMAAAAAAEhMGAQAAMD/b+8ObhsEggCKguQrpdGNy6Cb lJYCJsrBEjF2Ygwsu5P3jsiyAR+/ZhYAAEhMDAIAAAAAAEhMDAIAAAAAAEhMDAIAAAAAAEhMDAIA AAAAAEjk8+Paz59GDAIAAAAAAEhMDAIAAAAAAEhMDAIAAAAAAEjsRwy63yEHAAAAAABAOx61HpNB AAAAAAAAifURsXi6YZyWFwEAAAAAAKjSo4mgm4cx6EYUAgAAAAAAqNNvAWjusrgyM/8SYQgAAAAA AKA9zgwCAAAAAABozKtTQZ0YBAAAAAAAkJsYBAAAAAAAkJgYBAAAAAAAkJgYBAAAAAAAkNjFnwsA AAAAAFCvz49rv+Xm+ohYXHxmGKfXPwwAAAAAAMDbtkagG2viAAAAAAAAEhODAAAAAAAAElsVg/Ya RwIAAAAAAKAMk0EAAAAAAACJrY5BpoMAAAAAAADa0UfEppsdxmnbFwAAAAAAAPDDnsM5m2NQJwgB AAAAAAC87eitbLucGWR1HAAAAAAAQJ12iUEAAAAAAACsV2LgRgwCAAAAAABITAwCAAAAAABITAwC AAAAAABIbLcY9L3TrsReOwAAAAAAAF532ftdzYPQME6x+AAAAAAAAADFHLomzqQQAAAAAADAuQ4/ M0gQAgAAAAAAWCrVUA6PQQAAAAAAAJxHDAIAAAAAAEhMDAIAAAAAACis5DE7fUQsLh5hGKcyPwQA AAAAAFCpkhHoplgMekQgAgAAAAAAWndG4Fnj1DVxtb8cAAAAAACA1jkzCAAAAAAAIDExCAAAAAAA IDExCAAAAAAAILHTY5BzgwAAAAAAAI5TxWSQIAQAAAAAAHCMPiKqe7XDONV3UwAAAAAAAHdaGHip Mga9QjACAAAAAABKanXTWRVr4gAAAAAAADiGGAQAAAAAAJCYGAQAAAAAAPCHVlfEdWIQAAAAAABA bs3GoJYLHAAAAAAA0I7Wm0QfEYuLLRnGqe0HAAAAAAAAqpRlMKX5GHRPHAIAAAAAAH7z37aPpTsz yPo4AAAAAADgmf/YEdJNBq1higgAAAAAAHIwLPJE13VfdK3IMLrvUY0AAAAASUVORK5CYII=" transform="translate(2.093 1.031) scale(.0202)" overflow="visible"/></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/dpo-tigopesa.svg:1]: - The `<clipPath id="pi-dpo-tigopesa-SVGID_2_">` and the `<defs>` don't seem to be used. Can we remove it?
- I think we can remove the `<g clip-path="url(#SVGID_2_)">` and keep the `<image>` only. What are your thoughts?

File: app/assets/images/payment_icons/dpo-orangemoney.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...5iEMAgAAAAAAmJgwCAAAAAAAYGLCIAAAAAAAgIkJgwAAAAAAACYmDAIAAAAA AJiYMAgAAAAAAGBiwiAAAAAAAICJFYdBP378+OWnHwIAAAAAADA0M4P4/9u7g9SGYSCAohb4/ice mNJFoE3alKSybI3eW5rEsZzlZyQAAAAAAKAwMQgAAAAAAKAwMQgAAAAAAKCwlpkvry4iXv8SAAAA AAAA/7bve3vlHm/FoK+EIQAAAAAAgDFeDUFbjxi0CUIAAAAAAADdvRN+fuLMIAAAAAAAgMK6xKBe ZQoAAAAAAIC+TAYBAAAAAAAUJgYBAAAAAAAUJgYBAAAAAABcTM8jelpmPlzsISKOuTEAAAAAAEAx PePPvcNi0I0oBAAAAAAA8LsjQ9BmmzgAAAAAAIDzHB2CthExaMQiAAAAAAAA+JnJIAAAAAAAgMLE IAAAAAAAgMLEIAAAAAAAgMKGxCDnBgEAAAAAAHw3qp+0zHy4OEpEnPfjAAAAAAAAJxk5SHPqNnEm hgAAAAAAAI7lzCAAAAAAAICBRg/LiEEAAAAAAACDnLFrmhgEAAAAAAAwwFnH54hBAAAAAAAABzsr BH1qmflwcbSIOP8hAAAAAAAAOjszAt1cIgbdE4cAAAAAAIBZXSEAfXXJGHQjCgEAAAAAAFd3tfhz z5lBAAAAAAAAhV06Bl29pAEAAAAAAFydySAAAAAAAIDCxCAAAAAAAIDCxCAAAAAAAIDCWmZOt7qI mO+hAQAAAACA6e373mZbw5STQTO+aAAAAAAAYG6z9gnbxAEAAAAAAPxh5kGVaWOQ6SAAAAAAAIC/ mQwCAAAAAAAoTAwCAAAAAAB4YvbdyqaOQbaKAwAAAAAAeK5l5tMPzCwi6i4OAAAAAADopvIASukY 9BuRCAAAAAAA1rTirmPODAIAAAAAAChMDAIAAAAAAJaw4lTQJgYBAAAAAADUJgYBAAAAAAAUtmQM WnUMDAAAAAAAWE/LzGX/9ohYd/EAAAAAALCI1YdElo5B7xCQAAAAAABgnNVDTg9i0BsEIQAAAAAA OJYI1I8YdCDRCAAAAAAAvhN5Btu27QPYMig63QOFkAAAAABJRU5ErkJggg==" transform="matrix(.02021 0 0 .01992 2.165 1.248)" overflow="visible"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/dpo-orangemoney.svg:1]: Please remove `<style type="text/css"/>`

File: app/assets/images/payment_icons/dpo-mtnmobilemoney.svg [hunk 5]
```diff
@@ -0,0 +1 @@
...AAAAAAAQDAxCAAA AAAAINjNMci5QQAAAAAAAO0xGQQAAAAAABDs5hg0HoZy8SUAAAAAAABVMxkEAAAAAAAQbHfrrU1n Bv388e1tQujL5x8XvwMAAAAAAKAeD00G/fr9cvEdAAAAAAAA9bg7Bn19+d6ffhaEAAAAAAAA6vWh yaDzIAQAAAAAAMBzTEf73OrDa+IEIQAAAAAAgPr1pZSHL3I8DI//CQAAAAAAAO+6dxro1CwxqBOE AAAAAAAAZvVIADr14TVxAAAAAAAA1E8MAgAAAAAACCYGAQAAAAAAVGauFXGdGAQAAAAAAJBtthg0 Z6ECAAAAAABgHn0pZbFHOR6G5f4cAAAAAAAgxJJDN4uuiTMtBAAAAAAAcN3SPcWZQQAAAAAAACt5 xmDN4jHIdBAAAAAAAMB6Fo9Bzg0CAAAAAABYjzVxAAAAAAAAwayJAwAAAAAACLZ7xq2dBiFr4wAA AAAAAJ6nL2W9NiMMAQAAAAAA6dbeorbqmUFWyAEAAAAAACxr1RgEAAAAAADAssQgAAAAAACAYKvH IKviAAAAAACAVDV0EJNBAAAAAAAAwcQgAAAAAACABdSyHa0vpVx8uYbxMLxeyPRgps8AAAAAAAAt qe2InGpi0HsEIQAAAAAAoHa1xZ9z1sQBAAAAAAAEqzoG1V7SAAAAAAAAamcyCAAAAAAAIJgYBAAA AAAAEEwMAgAAAAAACNaXUpq7u/EwtHfRAAAAAABA83b7Y9/aPTQZgzpBCAAAAAAAeKIWI9Ck2TVx LT90AAAAAACAZ2n6zCBBCAAAAAAAWFrrPaLpGGRVHAAAAAAAwHUmgwAAAAAAAII1HYMAAAAAAACW lDCY0peSuWnNCjkAAAAAAOAep+Hnb2dI2VAWG4OuEYoAAAAAAGCbtngEjTVxAAAAAADAJmwxBHVi EAAAAAAAQLZNxqCtlj8AAAAAAGB7TAYBAAAAAAAE60spm32/42HY7s0DAAAAAMBGbH1j2KZj0EQU AgAAAACAPFuPQBMxaAZiEgAAAAAALEPQeZwYNCNRCAAAAAAAHicAzUsMejLBCAAAAACArRJ5VtB1 3R/tS2osGe3d2wAAAABJRU5ErkJggg==" transform="matrix(.02153 0 0 .02028 1 1)" overflow="visible"/></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/dpo-mtnmobilemoney.svg:1]: Same comments apply to `dpo-tigopesa.svg`

File: app/assets/images/payment_icons/dpo-airtelmoney.svg [hunk 6]
```diff
@@ -0,0 +1 @@
...EMAAAAAAACGZmYQAAAAAABAYMIgAAAAAACAwIRB AAAAAAAAgby++kcYBAAAAAAAEJgwCAAAAAAAILBDYdDrtCIAAAAq585qAAAC3UlEQVQAAADG9uXj 46PIBv7y7VuZDwIAAAAAAOCtvZN4LBMHAAAAAAAwmSOruQmDAAAAAAAAAhMGAQAAAAAABCYMAgAA AAAACEwYBAAAAAAAEJgwCAAAAAAAIDBhEAAAAAAAQGDCIAAAAAAAgMCEQQAAAAAAAIEJgwAAAAAA AAITBgEAAAAAAAQmDAIAAAAAAAhMGAQAAAAAADCRn79//3Jka798fHx8+mEpv3z7Vu/DAQAAAAAA gjsa/GypOjOoxAYCAAAAAACsqFTOYpk4AAAAAACAwIRBAAAAAAAAgQmDAAAAAAAAAqseBnlvEAAA AAAAQD9mBgEAAAAAAAT25ePjo9ne/fLtW7svAwAAAAAAmFTJldeahkGvhEMAAAAAAMDqar9yp+sy cd4nBAAAAAAArKxFVuKdQQAAAAAAAIEJgwAAAAAAAAITBgEAAAAAAAQmDAIAAAAAAOigxfuCbiOE Qa12FAAAAAAAYEVfPj4+htvtX759G2+jAAAAAAAALuoxSWbIMOhBKAQAAAAAAMyu9ypp3hkEAAAA AAAQ2NBhkPcJAQAAAAAAXGNmEAAAAAAAQCUjTHwRBgEAAAAAAAQmDAIAAAAAAKhglNfhfPn4+Pj0 w9H98u3bfBsNAAAAAACENUrws2XKmUEjFygAAAAAALCW0XMLy8QBAAAAAAAENm0YZHYQAAAAAADA e2YGAQAAAAAABCYMAgAAAAAAOGmGlcymDoMsFQcAAAAAAPQyS07x5ePj49MPo/jl27e4OwcAAAAA ADQ16ySV0GFQipAIAAAAAAB4FXVFMu8MAgAAAAAACEwYBAAAAAAAEJgwCAAAAAAAILAlw6Coa/4B AAAAAAC8MjMIAAAAAABYXuSJJF8+Pj4+/XAVv3z7tu7OAwAAAADA4lZZSWzpMGgvoREAAAAAAIzJ q2HeEwYdIBQCAAAAAID+BEDHCIMqEBoBAAAAAMA5gp7Cbrfb/wVE5ahohrqLWQAAAABJRU5ErkJg gg==" transform="matrix(.02161 0 0 .02023 1 1)" overflow="visible"/></g></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/dpo-airtelmoney.svg:1]: Same comments apply to `dpo-tigopesa.svg`

File: app/assets/images/payment_icons/cash.svg [hunk 7]
```diff
@@ -1 +1,9 @@
-<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" aria-labelledby="pi-cash" width="38" height="24" role="img"><title id="pi-cash">Cash</title><path fill="#98C74C" d="M38 21a3 3 0 0 1-3 3H3a3 3 0 0 1-3-3V3a3 3 0 0 1 3-3h32a3 3 0 0 1 3 3v18z"/><circle opacity=".2" cx="19" cy="12" r="8"/><circle opacity=".2" cx="32" cy="12" r="2"/><circle opacity=".2" cx="6" cy="12" r="2"/><path fill="#fff" d="M22.113 13.897c0-1.573-1.271-2.067-2.518-2.42V8.763c.65.118 1.218.588 1.68 1.105l.71-.797c-.578-.609-1.424-1.128-2.39-1.246V6.83h-1.034v.973c-1.408.154-2.413 1.036-2.413 2.275 0 1.461 1.2 1.907 2.413 2.244v3.024c-.898-.145-1.608-.759-2.053-1.277l-.62.945c.597.673 1.504 1.151 2.674 1.266v.89h1.034v-.881c1.819-.186 2.517-1.304 2.517-2.392zm-4.974-3.948c0-.684.627-1.11 1.422-1.223v2.459c-.797-.133-1.422-.596-1.422-1.236zm2.456 5.426v-2.762c.813.254 1.451.59 1.451 1.359 0 .565-.364 1.246-1.451 1.403z"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" aria-labelledby="pi-cash" width="38" height="24" role="img">
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/cash.svg:1]: Please run through svgo.

**Comment by linwor** [comment]
Replaced with new request

**Pull request closed without merging**